spec/cfn_nag_integration/cfn_nag_rds_instance_spec.rb

```diff
@@ -81,13 +81,13 @@
                 id: 'F23', type: Violation::FAILING_VIOLATION,
                 message: 'RDS instance master user password must not be a plaintext string or a Ref to a NoEcho Parameter with a Default value.',
                 logical_resource_ids: %w[BadDb1 BadDb2],
-                line_numbers: [14, 28]
+                line_numbers: [14, 29]
```

Comment:
In this test file, I only updated the line numbers. Please let me know if I should check for the new violation as well.